src/app/core/services/config/config.service.ts

```diff
@@ -29,6 +29,8 @@ import { SubjectConfigService } from './subject-config.service'
 
 @Injectable()
 export class ConfigService {
+  ATTRIBUTE_KEY_PREFIX = 'att-'
```

Comment:
I made this `att-` since `attribute-` is quite long and the key limit is only 24 chars.